tests/framework/framework.go

```diff
@@ -0,0 +1,242 @@
+package framework
+
+import (
+	"flag"
+	"fmt"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/golang/glog"
+	"k8s.io/api/core/v1"
+	apierrs "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+
+	cdiClientset "kubevirt.io/containerized-data-importer/pkg/client/clientset/versioned"
+)
+
+const (
+	NsCreateTime = 30 * time.Second
+	NsDeleteTime = 5 * time.Minute
+)
+
+// run-time flags
+var (
+	kubectlPath  *string
+	cdiInstallNs *string
+	kubeConfig   *string
+	master       *string
```

Comment:
Please add ocPath *string here as well, since the make functest passes it.

Reply:
Good point, I think I forgot to pass it. Basically the idea is that during the make manifest you can specify which NS you want it installed in, and that namespace is then passed to the tests so they check the right namespace for the cdi-sa and associated roles and bindings.

Reply:
ok, so we still want to handle the _installed-namespace_ flag, correct? I'll leave it in and you can create a pr to add it to the test cmd above?

Reply:
Yes because it has value if someone for whatever reason some installs cdi in a namespace that is not the default. Then the tests can still know which namespace cdi is installed in.

Reply:
ack

Reply:
@awels Are all the flags **required**? IOW, if a flag is omitted and the flag definition's default is "" is that an error/fail or ok and the code just uses ""?

Reply:
@jeffvance I am a little fuzzy on that, obviously cdiinstallns was missing, and setting a non empty string value make it so you didn't have to specify it. But if we omit ocPath, which currently we aren't using at all, it errors out. So I am a little fuzzy on that.

Reply:
Sounds like a plan to me.